Jenkinsfile

```diff
@@ -28,8 +28,8 @@ pipeline {
     }
     stage('Test') {
       steps {
-        sh 'docker-compose -p cardano-mainnet -f ./test/docker-compose-ci.yml up -d'
-        sh 'NODE_ENV=test TEST_MODE=e2e npx jest suite dataparity.test --ci'
```

Comment:
This test relies on external services, so cannot stop the build due to availability out of our control